JNWCollectionView/JNWCollectionViewCell.m

```diff
@@ -187,6 +187,8 @@ - (void)mouseUp:(NSEvent *)theEvent {
 }
 
 - (void)rightMouseDown:(NSEvent *)theEvent {
+  [super rightMouseDown:theEvent];
```

Comment:
I know the formatting is kind of a mess, but for any newer code I just am trying to keep it in line with most of the rest. So is there any chance you could tab this? 🎨